packages/edge/src/response.ts

```diff
@@ -20,3 +20,41 @@ export function json(data: any, init?: ResponseInit): Response {
   // @ts-expect-error This is not in lib/dom right now, and we can't augment it.
   return Response.json(data, init);
 }
+
+/**
+ * Builds a response for returning data based on promise that take many seconds to resolve.
+ * The response is returned immediately, but data is only written to it when the promise resolves.
+ *
+ * @param dataPromise Promise for data. Supported types of data are string and Uint Uint8Array
```

Comment:
maybe we don't need to tell the data types? docs are generated from typescript so they have a `Promise<string | Uint8Array>` mentioned in them automatically 😄 

```suggestion
 * @param dataPromise Promise for data to be sent as the response body
```